src/libecalc/presentation/exporter/dto/dtos.py

```diff
@@ -178,12 +206,32 @@ class AssetTSVPrognosis(EcalcResultBaseModel):
 
     tsv_prognoses: Dict[str, TSVPrognosis] = Field(default_factory=dict)
 
+    @property
+    def timesteps(self) -> Set[datetime]:
```

Comment:
```suggestion
    def common_timesteps(self) -> Set[datetime]:
```
? vs union, if that is suddenly needed (call it sth with intersection?)

Reply:
Yes, I wasn't sure, as all the timesteps should be the same for each installation right?

Reply:
not sure if it has or should, as I have experienced mismatches between mobile and fixed before ...